mobile-app/src/components/SearchBar.tsx

```diff
@@ -20,21 +24,32 @@ const styles = StyleSheet.create({
 
 // Component
 
-interface Props {
-  text: string;
-}
+//Redux goodness.
 
-export const SearchBar: React.FC<Props> = (props: Props) => {
-  const [query, setQuery] = useState('');
+const mapState = (state: RootState) => ({
+  search: state.artwork.searchQuery,
+});
+
+const mapDispatch = {
+  setSearchQuery: (query: string) => setSearchQuery(query),
+};
+const connector = connect(mapState, mapDispatch);
+type PropsFromRedux = ConnectedProps<typeof connector>;
+
+const SearchBar: React.FC<PropsFromRedux> = (props: PropsFromRedux) => {
 
   return (
     <View style={styles.container}>
       <TextInput
         style={styles.search}
-        placeholder={props.text}
-        value={query}
-        onChangeText={(newQuery) => setQuery(newQuery)}
+        placeholder={"Search"}
```

Comment:
```suggestion
        placeholder={"Search by title, author, or tag"}
```
Eh idk, might be too long for a placeholder